app/models/employment.rb

```diff
@@ -1,12 +1,22 @@
-class Employment < ApplicationRecord
-  belongs_to :assessment
+class Employment
+  include ActiveModel::Model
+  include ActiveModel::Attributes
 
-  has_many :employment_payments, dependent: :destroy
+  attribute :name, :string
+  attribute :client_id, :string
+  attribute :type, :string, default: "ApplicantEmployment"
+  attribute :receiving_only_statutory_sick_or_maternity_pay, :boolean, default: false
+  attribute :submission_date, :date
```

Comment:
the submission_date is only used for testing - probably ought to be a transient in the factory rather than a model property